data/mods/MindOverMatter/help_files.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+    "type": "help",
+    "order": 0,
+    "name": "Mind Over Matter: Basics",
+    "messages": [
+      "Welcome to Mind Over Matter, a psychic powers mod for Cataclysm.  This mod adds psionics to the game based on classical concepts of mental powers.  Incinerate zombies as a pyrokinetic, avoid bullets by peering into the near future as a clairsentient, recharge your car as an electrokinetic, run so fast you can run on water as a biokinetic, and more.  All these powers are available to you.",
+      "And to your enemies.  The player is not the only one with psionics.",
+      "Psionics are different in several ways than the magical powers in Magiclysm.  For one, all psionics require stamina to use rather than an external resource such as mana, meaning a full-charged psion is only a short breather away. Second, psionic powers scale with your intelligence stat and with your Nether Attunement (q.v.) as well as with the power level; this includes their maximum possible level, which is equal to your intelligence times 1.5.  Third, long-duration  powers must be concentrated on to be maintained, and negative effects like being injured, being electrocuted, or inhaling toxic smoke can cause you to lose your concentration.  Lose your concentration when you're levitating or maintaining a pyrokinetic shield against fire and you're in for a world of hurt and possibly the character creation screen.",
```

Comment:
```suggestion
      "Psionics are different in several ways than the magical powers in Magiclysm.  For one, all psionics require stamina to use rather than an external resource such as mana, meaning a full-charged psion is only a short breather away.  Second, psionic powers scale with your intelligence stat and with your Nether Attunement (q.v.) as well as with the power level; this includes their maximum possible level, which is equal to your intelligence times 1.5.  Third, long-duration  powers must be concentrated on to be maintained, and negative effects like being injured, being electrocuted, or inhaling toxic smoke can cause you to lose your concentration.  Lose your concentration when you're levitating or maintaining a pyrokinetic shield against fire and you're in for a world of hurt and possibly the character creation screen.",
```